.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           ruby-version: 2.7
           bundler-cache: true
-      - run: bundle exec rake rubocop
+      - run: bundle exec rubocop
```

Comment:
```suggestion
      - run: bundle exec rubocop
```
```suggestion
      - run: bundle exec rubocop --format github
```

Reply:
This workflow is about to get changed.. however I love the suggestion of `--format github`.

I'll implement it in the newer workflows.

Reply:
I just realized I didn't link to https://docs.rubocop.org/rubocop/1.22/formatters.html#github-actions-formatter, which I intended.